meshtastic/mesh_interface_runtime/flows.py

```
@@ -0,0 +1,561 @@
+"""Send flow methods for telemetry, position, traceroute, and waypoint operations."""
+
+from __future__ import annotations
+
+import logging
+import math
+import secrets
+from typing import TYPE_CHECKING, Any, Callable, Literal, TypeAlias
```

Ruff (0.15.7)

[warning] Import from `collections.abc` instead: `Callable`

Import from `collections.abc`

(UP035)